### ISSUE TITLE
Unable to boot on fresh install

### ISSUE THREAD

**Issue by AaronSadlerUK** [bjarnef/vendr-product-reviews#1]
Fresh install of the plugin from NuGet with Vendr 1.4.1 and I get the below error:

I have checked the database and I can see the table does not exist

```
There is already an object named 'PK_vendrReview' in the database.
Could not create constraint or index. See previous errors.
```

**Comment by bjarnef** [comment]
I couldn't reproduce the boot error when dropping the assembly to Vendr Demostore using Vendr v1.4.0 and SQL CE database.
Both tables were created although it also has create two contraints with name `PK_vendrReview`.

![image](https://user-images.githubusercontent.com/2919859/107409545-95af3800-6b0c-11eb-9d72-dd3fae0d3621.png)

**Comment by AaronSadlerUK** [comment]
I'm using SQL Developer edition if it makes any difference.

**Comment by bjarnef** [comment]
I dropped the constraints and the tables instead and installed via NuGet instead, but got both tables created + two constraints with same name. However it seems these must be unique though :)
https://stackoverflow.com/a/1397674

**Comment by AaronSadlerUK** [comment]
Ok, not sure then.

My PR fixed the issue, due to the PK clash.

**Comment by AaronSadlerUK** [comment]
@bjarnef Brilliant...

When will this version be out on NuGet?
I would like to remove my custom version containing this fix and use the NuGet version :)

**Comment by bjarnef** [comment]
@AaronSadlerUK I have just published a new v0.2.0 release on Our and NuGet 🤓 🚀 
Hopefully it will fix the issues you had 🤞

**Comment by AaronSadlerUK** [comment]
🥳🥳 Thanks!
